lib/Result.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+ * Result.cpp
+ *
+ *  Created on: 2021/09/25
+ *      Author: hommyo
+ */
+
+#include "Result.hpp"
+
+using namespace std;
+
+//全出力用関数
+void csv_all_result(const vector<Result>& results)
+{
+    FILE *csv = fopen("all_result.csv","w");
+
+    unsigned int results_len = results.size();
+    auto st = Statistics(results);
+
+    for(unsigned int i = 0; i < results_len; ++i)
+    {
+        double min_value = st.minimum();
+        double ave_value = st.average();
+
+        fprintf(csv,"%d,%f,%f,",i ,min_value ,ave_value);
+    }
+
+    fclose(csv);
+}
+//更新曲線
+void tprint_update_curve(const vector<double> value, const double number_of_updates, const char file_name)
+{
+    FILE *csv;
+    char fname[50];
+    sprintf(fname,"%d.csv",file_name);
```

Comment:
```suggestion
void Result::tprint_update_curve(const string& file_name) const
{
    FILE *csv;
```
拡張子まで含めて入力してもらいましょう
別に絶対ではないですが...